### PR TITLE
Make wireframes of MeshDebugPluginMaterial unshaded

### PR DESCRIPTION
**Pull request by alexchuber**
Before, shading was being applied to the wireframes of TRIANGLES, VERTICES, and TRIANGLES_VERTICES modes. This resulted in black wireframes turning grey-ish. These changes should fix the issue.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14094/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14094/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14094/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14094/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14094/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14094/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14094/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14094/merge#BCU1XR#0

**Review by carolhmj — APPROVED** [review]

File: packages/dev/core/src/Materials/meshDebugPluginMaterial.ts [hunk 1]
```diff
@@ -128,10 +128,11 @@ vec3 dbg_color = dbg_shadedDiffuseColor;
     gl_FragColor *= vec4(dbg_color, 1.);
 #else
     #if DBG_MODE != 6
-        gl_FragColor = vec4(dbg_applyShading(dbg_color), 1.);
+        gl_FragColor = vec4(dbg_applyShading(dbg_shadedDiffuseColor), 1.);
+        gl_FragColor *= vec4(dbg_color, 1.);
```
Comment (bghgary) [inline packages/dev/core/src/Materials/meshDebugPluginMaterial.ts:132]: Does this work?

```suggestion
        gl_FragColor = vec4(dbg_applyShading(dbg_shadedDiffuseColor) * dbg_color, 1.);
```

File: packages/dev/core/src/Materials/meshDebugPluginMaterial.ts [hunk 2]
```diff
@@ -128,10 +128,11 @@ vec3 dbg_color = dbg_shadedDiffuseColor;
     gl_FragColor *= vec4(dbg_color, 1.);
 #else
     #if DBG_MODE != 6
-        gl_FragColor = vec4(dbg_applyShading(dbg_color), 1.);
+        gl_FragColor = vec4(dbg_applyShading(dbg_shadedDiffuseColor), 1.);
+        gl_FragColor *= vec4(dbg_color, 1.);
     #else
         gl_FragColor = vec4(dbg_color, 1.);
-    #endif                
+    #endif
```
Comment (bghgary) [inline packages/dev/core/src/Materials/meshDebugPluginMaterial.ts:135]: nit
```suggestion
    #endif
```

**Review by bghgary — APPROVED** [review]

File: packages/dev/core/src/Materials/meshDebugPluginMaterial.ts (hunk 1 above)
Comment (alexchuber) [reply packages/dev/core/src/Materials/meshDebugPluginMaterial.ts]: Yes!

**Review by Popov72 — APPROVED** [review]

**Pull request merged**